test/spec/Feature/Query/UpdateSpec.hs

```diff
@@ -607,8 +619,186 @@ spec = do
           , { "id": 3, "name": "item-3" }
           ]|]
 
-      request methodPost "/rpc/reset_limited_items"
+      request methodPost "/rpc/reset_items_tables"
         [("Prefer", "tx=commit")]
         [json| {"tbl_name": "limited_update_items_no_pk"} |]
         `shouldRespondWith` ""
         { matchStatus  = 204 }
+
+  context "bulk updates" $ do
+    it "can update tables with simple pk" $ do
+      get "/bulk_update_items"
+        `shouldRespondWith`
+          [json|[
+            { "id": 1, "name": "item-1", "observation": null }
+          , { "id": 2, "name": "item-2", "observation": null }
+          , { "id": 3, "name": "item-3", "observation": null }
+          ]|]
+
+      request methodPatch "/bulk_update_items"
+          [("Prefer", "tx=commit")]
+          [json|[
+            { "id": 1, "name": "item-1 - 1st", "observation": "Lost item" }
+          , { "id": 3, "name": "item-3 - 3rd", "observation": null }
+          ]|]
+        `shouldRespondWith`
+          ""
+          { matchStatus  = 204
+          , matchHeaders = [ matchHeaderAbsent hContentType
+                           , "Content-Range" <:> "0-1/*"
+                           , "Preference-Applied" <:> "tx=commit" ]
+          }
```

Comment:
Using the pks as filters without the option to reference other key in the body is too arbitrary. We need to be able to refer to every key, like [bulk delete](https://github.com/PostgREST/postgrest/pull/2355#issue-1291906948) does.

For this, we'd need [underscore operators](https://github.com/PostgREST/postgrest/issues/2125#issuecomment-1200054135).